tests/github-action-helper.sh

```diff
@@ -33,12 +33,35 @@ deploy_rook() {
   curl https://raw.githubusercontent.com/rook/rook/master/deploy/examples/cluster-test.yaml -o cluster-test.yaml
   sed -i "s|#deviceFilter:|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" cluster-test.yaml
   kubectl create -f cluster-test.yaml
+  wait_for_pod_to_be_ready_state_default
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/storageclass-test.yaml
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/pvc.yaml
 }
 
-# wait_for_pod_to_be_ready_state check for operator pod to in ready state
-wait_for_pod_to_be_ready_state() {
+deploy_rook_in_custom_namespace() {
+  kubectl create namespace test-operator # creating namespace manually beacuse rook common.yaml create one namespace and here we need 2
```

Comment:
```suggestion
  kubectl create namespace test-operator # creating namespace manually because rook common.yaml create one namespace and here we need 2
```